app/messages.py

```diff
@@ -9,6 +10,7 @@
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
     "message": "Field available_to_mentor" " is not valid."
 }
+PASSWORD_INPUT_BY_USER_HAS_INVALID_LENGTH={"message": "The password field has to be longer than 8 characters and shorter than 64 characters."}
```

Comment:
```suggestion
PASSWORD_INPUT_BY_USER_HAS_INVALID_LENGTH={"message": f"The password field has to be longer than {PASSWORD_MIN_LENGTH} characters and shorter than {PASSWORD_MAX_LENGTH} characters."}
```
Have you thought about doing this? @Vuyanzi

Reply:
No i didn't. This actually a good idea. So it will be using values as set in the validations. Thank you, let me import the variables